### PR TITLE
add a Unit conversion method

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

File: src/main/java/com/arpnetworking/metrics/Units.java [hunk 1]
```diff
@@ -33,6 +36,33 @@
...
 public final class Units {
 
+    /**
+     * Converts a {@link TimeUnit} to a {@link Unit}.
+     *
+     * @param timeUnit The {@link TimeUnit} to convert.
+     * @return A {@link Unit} representing the {@link TimeUnit}.
+     */
+    @Nullable
+    public static Unit from(@Nullable final TimeUnit timeUnit) {
```
Comment (vjkoskela) [inline src/main/java/com/arpnetworking/metrics/Units.java]: Should we provide a variant that returns Optional?

File: pom.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
   <name>Metrics Client (Java)</name>
   <description>Client library in Java for metrics collection and publication.</description>
   <url>https://github.com/ArpNetworking/metrics-client-java</url>
-  <version>0.6.1-SNAPSHOT</version>
+  <version>0.7.0-SNAPSHOT</version>
```
Comment (vjkoskela) [inline pom.xml]: Where's the deprecation of the method we want to remove?

File: src/main/java/com/arpnetworking/metrics/Units.java (hunk 1 above)
Comment (BrandonArp) [inline src/main/java/com/arpnetworking/metrics/Units.java]: Sure, easy.

File: pom.xml (hunk 2 above)
Comment (BrandonArp) [inline pom.xml]: Got it now.

**Pull request merged**